src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/AzureTarget.tsx

```diff
@@ -0,0 +1,312 @@
+ * @param setTenantId a function to update the tenantId
+ * @param subscriptionId a value either computed via the sources if a sourceId
+ * was selected or manually entered by the user
+ * @param setSubscriptionId a function to update the subscriptionId
+ * @param resourceGroup holds a resource group the user either selected
+ * in a set of possibilities or entered manually
+ * @param setResourceGroup a function to update the
+ * ResourceGroup
+ * @param resourceGroups a list of resource groups computed via the sources if a
+ * sourceId was selected that contains choices for the user to select
+ * the resourceGroup.
+ */
+const AzureTarget = ({
+  manual,
+  setManual,
+  isErrorFetchingDetails,
+  sourceId,
+  setSourceId,
+  tenantId,
+  setTenantId,
+  subscriptionId,
+  setSubscriptionId,
+  resourceGroup,
+  setResourceGroup,
+  resourceGroups,
+}: AzureTargetPropTypes) => {
+  const [isErrorFetchingSources, setIsErrorFetchingSources] = useState(false);
+  const clearState = () => {
+    // when the user switches from manual to automatic, reset everything
+    setSourceId(0);
+    setTenantId('');
+    setResourceGroup('');
+    setSubscriptionId('');
+  };
+
+  return (
+    <>
+      <Form>
+        <Title headingLevel="h2">Target environment - Microsoft Azure</Title>
+        <TextContent>
+          <Text>
+            Upon build, Image Builder sends the image to the selected authorized
+            Azure account. The image will be uploaded to the resource group in
+            the subscription you specify.
+          </Text>
+          <Text>
+            To authorize Image Builder to push images to Microsoft Azure, the
+            account owner must configure Image Builder as an authorized
+            application for a specific tenant ID and give it the role of
+            &quot;Contributor&quot; for the resource group you want to upload
+            to. This applies even when defining target by Source selection.
+            <br />
+            <Button
+              component="a"
+              target="_blank"
+              variant="link"
+              icon={<ExternalLinkAltIcon />}
+              iconPosition="right"
+              isInline
+              href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow"
+            >
+              Learn more about OAuth 2.0
+            </Button>
+          </Text>
+        </TextContent>
+        <FormGroup label="Share method:" isRequired>
+          <Radio
+            name="azure-share-source"
+            id="azure-share-source"
+            label="Use an account configured from Sources."
+            description={
+              'Use a configured source to launch environments directly from the console.'
+            }
+            checked={!manual}
+            isChecked={!manual}
+            onClick={() => {
+              setManual(false);
+              clearState();
+            }}
+          />
```

Comment:
A bit more space between the radios would be nice again. We could probably make the gap a bit wider globally?

Reply:
I don't know, I can add a `<br />` :shrug: